pkgs/development/interpreters/python/python-packages-base.nix

```diff
@@ -61,7 +61,43 @@ let
 
   removePythonPrefix = lib.removePrefix namePrefix;
 
-  # Convert derivation to a Python module.
+  /**
+    A function that takes a Python package and returns a new derivation that can be used as a Python module
+
+    # Inputs
+
+    `drv` (derivation)
+    : The derivation to convert to a python module
+
+    # Type
+    ```
+    derivation -> derivation
+    ```
+
+    # Example
+    :::{.example}
+    ## Using `toPythonModule` to provide `home-assistant` as a library to a python script.
+    ```python
+    # main.py
+    import homeassistant
+
+    print("Hello ", homeassistant)
+    ```
```

Comment:
This can be replaced by a simple ``python -c "import opencv"`` which will error when the module is not found